### ISSUE TITLE
Create scripts to run test patients and searches on wmda search and match

### ISSUE THREAD

**Issue by zabeen** [Anthony-Nolan/Atlas#950]
@mmelchers has provided postman collection on Atlas teams.

Need to check:
- how patient data can be loaded into request
- how to track Atlas search request ID and test patient ID for future results analysis

**Comment by zabeen** [comment]
Priority test cases are 10, 11 and 12

**Comment by IgorKupreychik** [comment]
Test searches for cases 10, 11 and 12 have been run, files with WMDA patient ids and search ids have been posted in Teams 'General' channel.

**Comment by zabeen** [comment]
@IgorKupreychik could you please write a summary of the solution you created to run the test searches? We should also upload the various scripts to WMDA github, so they are available for the next time we want to run something. Then we can close this ticket.

**Comment by IgorKupreychik** [comment]
I've done the following to run those searches:

1.  Updated script for WMDA patient creation: 

- Postman treats empty values from csv as empty strings, but, unfortunately, SearchAndMatch doesn't allow empty strings for HLA values (it expects NULLs instead), so I had to re-write the script and build body dynamically for each request (csv files have different format/number of columns, so I had to adjust it almost for each file).
- Updated 'Tests' section to send request to another server 'localhost:3000' with wmdaId (made same updates for 'search' requests to send wmda searchId)

2. Created a little script and hosted it as Node.js server to listen on 'localhost:3000' and write received results in to the file (result.csv). Dependencies: node.js, npm, express. 


Considering the difference in format of the input files, the fact that initially it's been discussed like 'one-time' run stuff and therefore there was not much requirements provided and the current solution is really custom and requires manual intervention for each test run, I don't think it would help a lot if we post it somewhere though if we really need -  I can share it.
